test/LieGroupsTestSuite.jl/LieGroupsTestSuite.jl

```diff
@@ -398,15 +398,15 @@ function test_exp_log(
                 exp!(G, k2, e, X)
                 @test isapprox(G, k1, k2)
             end
-            @test is_point(G, k1)
+            @test is_point(G, k1; error=:warn)
```

Comment:
I would maybe even error these, a failing test would print an error already, this error would give more information?

Reply:
A failing test doesn't tell why the point was incorrect without this.

Reply:
until now I set that to `error=:error`, if you prefer to warn only (and the failing test to error), we could change that everywhere, sure.

Reply:
I can change it to error.